device/package.json

```diff
@@ -27,6 +27,7 @@
     "native"
   ],
   "scripts": {
+    "test": "uvu -r esm -r ts-node/register src/__tests__",
```

Comment:
Yeah, I added `esm` since ts-node doesn't have esm support: https://github.com/TypeStrong/ts-node/issues/1007 It should be fine for our use case.

Alternatively, we could
- run `build` as a pretest script
- write the tests in javascript
- use node's native module support instead of esm